pprint/src/main/scala/ammonite/pprint/Core.scala

```diff
@@ -7,12 +7,15 @@ import acyclic.file
  *
  * @param maxWidth Controls how far to the right a line will go before
  *                 it tries to wrap
+ * @param maxHeight Controls how many lines can be printed at once. 
+ *                  Will print all lines if set to 0
  * @param depth How much the current item being printed should be indented
  * @param renames A map used to rename things to more common names, e.g.
  *                renamig `WrappedArray` to `Array` or getting rid of
  *                TupleN *
  */
 case class Config(maxWidth: Int = 100,
+                  maxHeight: Int = 15,
```

Comment:
I'd make this `0`, so by default it'll print everything, and set it to `15` only for the config used in the REPL. In theory someone using this for non-REPL stuff such as logging would want the whole output